test.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const { cognito } = require('./index')
+
+cognito.getAllUsers()
+  .then((users) => {
+    console.log(users.length)
+  })
```

Comment:
I'm assuming this is not meant to be here

Reply:
Yes, good point